input/docs/handbook/commands/index.md

```diff
@@ -15,9 +15,8 @@ A `ReactiveCommand` is created using static factory methods which allows you to
 // A synchronous command taking a parameter and returning nothing.
 // The Unit type is often used to denote the successfull completion
 // of a void-returning method (C#) or a sub procedure (VB).
-var command = ReactiveCommand.Create<int, Unit>(
-    integer => Console.WriteLine(integer)
-);
+ReactiveCommand<int,Unit> command = ReactiveCommand.Create<int>(
```

Comment:
Which ReactiveUI version you are using? Afair, the syntax `.Create<TIn>` was deprecated a long time ago, and now one needs to specify both `TIn` and `TOut` types explicitly.

Reply:
I'm using the latest version (11.1.6) and I get the following error:
`Cannot implicitly convert type 'void' to 'System.Reactive.Unit'`
Removing `Unit` in `TOut` fixes the error and I changed `var` to the whole command to make that clear. Is there another way to fix this error?

Reply:
Hmm, thank you for pointing this out and submitting a fix! Looks like I missed that we actually have the [`Create<TParam, TResult>`](https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/ReactiveCommand/ReactiveCommand.cs#L220), [`Create<TParam>`](https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/ReactiveCommand/ReactiveCommand.cs#L175) and [`Create<TResult>`](https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/ReactiveCommand/ReactiveCommand.cs#L135) factory method overloads now, this seems to have been changed in one of the latest ReactiveUI releases. The `api/reactiveui/reactivecommand` stuff gets generated from [ReactiveCommand API docs](https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/ReactiveCommand/ReactiveCommand.cs#L42) btw. Thanks again for submitting the fix!

Reply:
Been that way for a while @worldbeater. We deprecated the non generic ReactiveCommand class a while back.